source/contracts/signatures.md

```diff
@@ -20,18 +20,33 @@ To be able to have the ECDSA signature without the need of additional data we us
 
 `r`, `s` and `v` are the required parts of the ECDSA signature to recover the signer.
 
+---------
+### `eth_sign` signature
+`signature type > 30`
+
+To be able to use `eth_sign` we need to take the parameters `r`, `s` and `v` from calling `eth_sign` and set `v = v + 4`
+
+#### Constant part:
+`{32-bytes r}{32-bytes s}{1-byte v}`
+
+`r`, `s` and `v` are the required parts of the ECDSA signature to recover the signer.
+`v` will be substracted by `4` to calculate the signature.
+
 ---
 ### Contract Signature (EIP-1271)
 `signature type == 0`
 
 #### Constant part:
-`{32-bytes signature verifier}{32-bytes data position}`
+`{32-bytes signature verifier}{32-bytes data position}{1-byte signature type}`
```

Comment:
I left these out here since they don't contain information for this signature type (e.g. the signature type byte is always 0)

Reply:
Ok, I added that because it wasn't clear in case someone wants to craft a signature and I think it doesn't harm, do you think we should keep that out?

Reply:
no, I think it is good to know (also in the other cases), I am just not sure if we should mention that it is constant (same in the other cases, maybe mention that `ignored` should be 32 0-bytes normally) .... but this should work so no need to change it really ... if somebody gets confused later we can change it later :P